src/apis/Stocker.ts

```diff
@@ -1,39 +1,85 @@
 import { Amplify, API } from 'aws-amplify';
 import { DateTime } from 'luxon';
 
+import { Commodity, Price } from '@/book/entities';
+import { getMainCurrency } from '@/lib/queries';
+import { toAmountWithScale } from '@/helpers/number';
+
 import awsExports from '../aws-exports';
 
 Amplify.configure(awsExports);
 
 const API_NAME = 'stocker';
 
-export async function getPrices(tickers: string[]): Promise<{ [key: string]:LiveSummary; }> {
-  const options = {
-    queryStringParameters: {
-      ids: [tickers].toString(),
-    },
-  };
-  const resp = await API.get(API_NAME, '/api/prices', options);
+/**
+ * Connect to Stocker API and retrieve current prices for
+ * all currency pairs and commodities.
+ */
+export async function insertTodayPrices(): Promise<void> {
```

Comment:
This is very ugly but I want it this way, self-contained so we decouple Stocker as much as possible from the rest of the code. This goes in line of having in the future "extensions" or "plugins" that scan for prices